app/config/services.php

```diff
@@ -18,6 +23,24 @@
  */
 
 $container = new ContainerBuilder();
+
+$loader = new FilesystemLoader(__DIR__.'/../views/%name%');
+$session = new Symfony\Component\HttpFoundation\Session\Session();
+$session->start();
+
+$templating = new PhpEngine(new TemplateNameParser(), $loader);
+
+$faker = Factory::create();
+
+$container->set('templating', $templating);
+
+// Zet een service - bij instantie
+$container->set('faker', $faker);
+$container->set('session' , $session);
+
+// Registeer een service - bij naam
 $container->register('foo', 'Frissr\Volunteer\Entity\Foo');
```

Comment:
Uit mijn begrip, zet de set methode van de container een object instantie in de container (zodat je die eruit kan halen wanneer je wilt). Maar hoe zit het met register; wanneer doe je dit, waarom doe je dit? Het lijkt alsof je een class stopt in de container ipv instantie met de naam 'foo'?